internal/stream/stream.go

```diff
@@ -184,9 +187,73 @@ func (s *Stream) StartReader(reader Reader) {
 
 	sr.start()
 
-	for _, sm := range s.streamMedias {
+	for m, sm := range s.streamMedias {
 		for _, sf := range sm.formats {
 			sf.startReader(sr)
+			if m.Type == description.MediaTypeVideo {
+				cb := sf.runningReaders[sr]
+				if cb == nil {
+					continue
+				}
+
+				cu := s.CachedUnits[:0]
+				for _, v := range s.CachedUnits {
+					if v.(*unit.H264).AU != nil {
+						cu = append(cu, v)
+					}
+				}
+				if len(cu) == 0 {
+					continue
+				}
+
+				playbackFPS := int64(200)
```

Comment:
This would definitely benefit from a comment explaining why we're playing at 200FPS. ;)